MSAL/src/public/MSALAuthority.h

```diff
@@ -31,12 +31,13 @@
 
 @interface MSALAuthority : NSObject
 
-@property (readonly, nonnull) NSURL *url;
+NS_ASSUME_NONNULL_BEGIN
+@property (readonly) NSURL *url;
 
-- (instancetype _Nullable )init NS_UNAVAILABLE;
-+ (instancetype _Nullable )new NS_UNAVAILABLE;
+- (instancetype _Null_unspecified )init NS_UNAVAILABLE;
```

Comment:
why _Null_unspecified?

Reply:
The init() method defined in the NSObject class does no initialization; it simply returns self. In terms of nullability, callers can assume that the NSObject implementation of init() does not return nil.